packages/apollos-api/src/data/feature-feeds/index.js

```diff
@@ -0,0 +1,17 @@
+import { FeatureFeed as coreFeatureFeed } from '@apollosproject/data-connector-rock';
+
+const { schema, resolver } = coreFeatureFeed;
+
+class FeatureFeed extends coreFeatureFeed.dataSource {
+  baseGetFeed = this.getFeed;
+
+  // this will also embed campus into the feature feed id
+  getFeed = async ({ type, args }) => {
+    const {
+      campusId,
+    } = await this.context.dataSources.Person.getCurrentUserCampusId();
+    return this.baseGetFeed({ type, args: { ...args, campusId } });
```

Comment:
@vinnyjth thought i was doing that here

Reply:
You had the code perfect for the `FeedIds`, but the features themselves all shared the same IDs. Meant that the feed itself would rerender, but the Features would each have the same ids as before and thus wouldn't decide to rerender.

Reply:
oh like if you changed the campus?

Reply:
Yah, totally. Worked fine in all other situations.